scripts/sv/manage_sv_pipeline.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# This script manages the SV discovery pipeline by calling other scripts
+# a) it creates a Google Dataproc cluster used for running the GATK-SV
+#    pipeline (call create_cluster.sh)
+# b) it runs the analysis (call runWholePipeline.sh)
+# c) it copies results to a datetime and git-version stamped directory
+#    on GCS, along with logged console output (call copy_sv_results.sh)
+# d) it shuts down the cluster
+
+# terminate script on error or if a command fails before piping to another command
+set -eu
+set -o pipefail
+
+if [[ "$#" -lt 4 ]]; then
+    echo -e \
+"Please provide:
+  [1] local directory of GATK build (required)
+  [2] project name (required)
+  [3] GCS path to indexed BAM (required)
+  [4] GCS path to reference fasta (required)
+      OPTIONAL arguments:
+  [5] path to initialization script (local or GCS, defaults to
+      ${GATK_DIR}/scripts/sv/default_init.sh if omitted or empty)
+  [6] GCS username (defaults to local username if omitted or empty)
+  [*] additional arguments to pass to
+      StructuralVariationDiscoveryPipelineSpark"
+    exit 1
+fi
+
+# init variables that MUST be defined
+GATK_DIR="$1"
+PROJECT_NAME="$2"
+GCS_BAM="$3"
+GCS_REFERENCE_FASTA="$4"
+
+# init variables with optional overrides / defaults
+# passed value -> overrides system value -> overrides default value
+INIT_SCRIPT=${5:-${INIT_SCRIPT:-"${GATK_DIR}/scripts/sv/default_init.sh"}}
+GCS_USER=${6:-${GCS_USER:-${USER}}}
+shift 6
+SV_ARGS=${*:-${SV_ARGS:-""}} && SV_ARGS=${SV_ARGS:+" ${SV_ARGS}"}
+
+# add GATK SV scripts to PATH
+PATH="${GATK_DIR}/scripts/sv:${PATH}"
+
+# configure caching .jar files
+export GATK_GCS_STAGING="gs://${PROJECT_NAME}/${GCS_USER}/staging/"
```

Comment:
I would prefer a check if an environment variable has been set or not, because a user may already have this set in my shell profile ( I personally do), and preferably would like the jars being cached there.